Makefile

```diff
@@ -35,114 +27,62 @@ PORT ?= 6013
 help:
 	@cat $(MAKEFILE_LIST) | grep -E '^[a-zA-Z_-]+:.*?## .*$$' | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-.PHONY: preview
-preview: ## Set environment to preview
-	$(eval export CF_SPACE=preview)
-	@true
-
-.PHONY: staging
-staging: ## Set environment to staging
-	$(eval export CF_SPACE=staging)
-	@true
-
-.PHONY: production
-production: ## Set environment to production
-	$(eval export CF_SPACE=production)
-	@true
-
 # ---- LOCAL FUNCTIONS ---- #
 # should only call these from inside docker or this makefile
 
 .PHONY: _generate-version-file
 _generate-version-file:
 	@echo -e "__commit__ = \"${GIT_COMMIT}\"\n__time__ = \"${DATE}\"" > ${APP_VERSION_FILE}
 
-.PHONY: _run
-_run:
-	# since we're inside docker container, assume the dependencies are already run
-	./scripts/run_app.sh ${PORT}
-
-.PHONY: _run-celery
-_run-celery:
-	# since we're inside docker container, assume the dependencies are already run
-	./scripts/run_celery.sh
-
-.PHONY: _test
-_test:
-	# since we're inside docker container, assume the dependencies are already run
-	./scripts/run_tests.sh
-
-.PHONY: _single_test
-_single_test:
-	pytest -k ${test_name}
-
-define run_docker_container
-	docker run -it --rm \
-		--name "${DOCKER_CONTAINER_PREFIX}-${1}" \
-		-e NOTIFY_APP_NAME=${NOTIFY_APP_NAME} \
-		-e GIT_COMMIT=${GIT_COMMIT} \
-		-e TEMPLATE_PREVIEW_INTERNAL_SECRETS=${TEMPLATE_PREVIEW_INTERNAL_SECRETS} \
-		-e STATSD_ENABLED= \
-		-e NOTIFY_ENVIRONMENT=${CF_SPACE} \
-		-e AWS_ACCESS_KEY_ID=$${AWS_ACCESS_KEY_ID:-$$(aws configure get aws_access_key_id)} \
-		-e AWS_SECRET_ACCESS_KEY=$${AWS_SECRET_ACCESS_KEY:-$$(aws configure get aws_secret_access_key)} \
-		-e DANGEROUS_SALT=${DANGEROUS_SALT} \
-		-e SECRET_KEY=${SECRET_KEY} \
-		-e NOTIFICATION_QUEUE_PREFIX=${NOTIFICATION_QUEUE_PREFIX} \
-		${3} \
-		${DOCKER_IMAGE_NAME} \
-		${2}
-endef
-
-
 # ---- DOCKER COMMANDS ---- #
 
-.PHONY: run-with-docker
-run-with-docker: prepare-docker-build-image ## Build inside a Docker container
-	$(call run_docker_container,build, make _run, -p ${PORT}:${PORT})
-
-.PHONY: run-celery-with-docker
-run-celery-with-docker: prepare-docker-build-image ## Build Celery app inside a Docker container
-	$(if ${NOTIFICATION_QUEUE_PREFIX},,$(error Must specify NOTIFICATION_QUEUE_PREFIX))
-	$(call run_docker_container,celery-build, make _run-celery)
+.PHONY: bootstrap
+bootstrap: _generate-version-file
+	mkdir -p log # manually create directory to avoid permission issues
+	docker build -f docker/Dockerfile --target test -t notifications-template-preview .
 
-.PHONY: bash-with-docker
-bash-with-docker: prepare-docker-test-build-image ## Build inside a Docker container
-	$(call run_docker_container,build, bash)
+.PHONY: run-flask
+run-flask:
```

Comment:
Can we add the help comments (adding ## run flask in docker container), please?

Reply:
Ahh...after reading your comment I realised these are used for `make help`. Sure, I've amended the 5th commit with some comments. Does that look OK now?